app/views/admin/teachers/_teacher.html.erb

```diff
@@ -0,0 +1,9 @@
+  <tr>
+    <td><%= teacher.user.full_name %></td>
+    <td><%= teacher.user.address %></td>
+    <td><%= teacher.user.email %></td>
+    <td><%= teacher.subject.name %></td>
```

Comment:
add `subject_name` method to the teacher to return subject's name

Reply:
added.

Reply:
and wrote test